### ISSUE TITLE
A way to recover passwords on a vault

### ISSUE THREAD

**Issue by Kormut** [deckerst/aves#1188]
**Is your feature request related to a problem? Please describe.**
It's kind of related to a bug. When your Vaults are protected by System Lock and you remove the system lock on your phone, there is no way to add pictures or open your vaults. It just gives an error. I temporarily made a vault with a password, forgot (or made a mistake while making it) my password. But there is no way to reset or recover my password. 

**Describe the solution you'd like**
A way to recover or reset the password I created on a vault. 

**Describe alternatives you've considered**
Maybe through email? But that would be hard to make probably. Otherwise maybe a question like *name of first pet?* that kind off stuff.

**Comment by deckerst** [comment]
It's not Aves responsibility to manage your passwords. You could use a password manager instead of manually creating and remembering them.

**Comment by Kormut** [comment]
Completely missed the point and the bug I explained but okay.
I use the same password for everything and it worked fine, until I reinstalled my System Lock on my phone, from that point on it gives an error when I type the password.
Maybe should have put this under bug but I suspect the answer I would have gotten to be the same.

**Comment by deckerst** [comment]
Oh I thought you had 2 separate issues:
Scenario 1: you were using system lock (fingerprint, pattern, whatever), you set up a vault with it, you removed system lock, vault can't be open,
Scenario 2: you set up a vault with a custom password (not a system lock password), and you forgot it.

My comment was addressing 2), as 1) seemed obvious.

> I reinstalled my System Lock on my phone

Not sure I understand this. There's nothing to install. Do you mean you reconfigured the system lock? And you were using a password (not fingerprint, not pattern) before and after, right?

**Comment by Kormut** [comment]
Yes i reconfigured my fingerprint in my phone settings and from that moment on my normal vaults were working again, but the one I locked with a password keeps saying I put in the wrong password even though it was working fine before.
My fault for hiding my personal bug as a problem for people forgetting their passwords, I was trying to kill 2 birds with 1 stone here.

**Comment by deckerst** [comment]
Let me rephrase to see whether I understood ^^

The password-protected vault won't open with what you believe is the correct password, and you suspect that it is because you changed the system lock in the meantime, even though that password-protected vault is not relying on the system lock for protection.

I'll see whether I can reproduce the issue by following that sequence.

**Comment by Kormut** [comment]
Yes exactly, I just used my standard password so I am 100% certain I got the right one. 
I use the Nothing Phone (2) btw so not a really common phone. It runs on android 14 though.

**Comment by deckerst** [comment]
I've just tried it on my device and everything works as expected.

What I've tried:
1) I created vault A with the system lock (pattern / fingerprint)
2) I created vault B with the custom password lock
3) I put images in vault A and in vault B
4) I verified I can lock and unlock vaults A and B
5) I removed the system lock by going to the Android security settings, then selecting "None" as a screen lock
6) in Aves, I can no longer unlock vault A (not sure whether it's a problem, cf later step), but I can unlock B just fine
7) I set up the system lock by going to the Android security settings, then selecting "Pattern" as a screen lock
8) in Aves, I can unlock vault A again, and I can still unlock B

Could you follow this and let me know whether you can or cannot unlock vaults A and B?
